Class/C-Array-Methods/exercise6.js

```diff
@@ -6,10 +6,21 @@ let scoresList = [45, 83, 15, 8, 62, 60, 79];
 // It should return true if the score is 60 or above
 // otherwise return false
 
+function hasPassed (score){
+    console.log(score);

```

Comment:
Using `console.log` for debugging can be really helpful so I'm glad to see you doing it here. Make sure though that if you do that you tidy up and comment or remove it once you're done.